ui/main.cpp

```diff
@@ -87,6 +87,13 @@ void setDarkMode(QApplication &app)
 int main(int argc, char *argv[])
 {
     // Check number of arguments
+    bool exit_after_load = false;
```

Comment:
Consider adding absl flag parsing.

Reply:
yes, that can be done later so that other flags is using absl as well.